OneCommunityGlobal/HighestGoodNetworkApp
# Shantanu unit test pie chart

# Description
Wrote unit test cases for the Pie Chart component

![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/85691749/132ef167-b1be-4d22-be63-0e33e535dc2b)
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/85691749/8efb7e0f-678e-4625-bed0-684f8487a2b5)


## Main changes explained:
- Added a null safety zero in the file
…

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. run in tersimal: npm test PieChart.test.jsx
4. check if everything passed

## Screenshots or videos of changes:

## Note:
Include the information the reviewers need to know.
The Map function in the component do not have a key property
